policy/modules/kernel/storage.if

```diff
@@ -310,6 +310,30 @@ interface(`storage_dev_filetrans_fixed_disk',`
 	dev_filetrans($1, fixed_disk_device_t, blk_file, $2)
 ')
 
+########################################
+## <summary>
+##	Create char devices in /dev with the fixed disk type
+##	via an automatic type transition.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <param name="filename" optional="true">
+##	<summary>
+##	Optional filename of the char device to be created
+##	</summary>
+## </param>
+#
+interface(`storage_dev_filetrans_fixed_disk_control',`
```

Comment:
Please remove `_control` from the name and make the obj class a parameter.

Reply:
There was already an interface without the _control. Following the example of dev_filetrans_cardmgr() I made it take both chr_file and blk_file,